requirements.in

```diff
@@ -18,7 +18,7 @@ django-ckeditor==5.9.0
 django-cors-headers
 django-image-cropping
 easy_thumbnails
-git+https://github.com/Tampere/django-tamusers.git@d4f78bfb1cbff278d02b1bcccd330393f8e78d86#egg=django-tamusers
+git+https://github.com/Tampere/django-tamusers.git@9eb5f84171b1a2f525f28de408171608dd122574#egg=django-tamusers
```

Comment:
Just want to check this is using the correct django-tamusers version

Reply:
I think it's incorrect actually. Must've messed this up when fixing the conflicts.

Reply:
No, this is actually correct now. `requirements.in` still had the old commit pinned previously.